website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,144 @@
+---
+layout: docs
+page_title: Downgrade from Consul Enterprise to the community edition
+description: >-
+  Learn how to downgrade your installation of Consul Enterprise to the free Consul community edition (CE)
+---
+
+# Downgrade from Consul Enterprise to the community edition
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
```

Comment:
Just a thought; instead of specifically instructing to download the version, I think we can follow our standard upgrade guide format and say _install the required CE version_; and the installation specifics should be left to the user.

ref: https://developer.hashicorp.com/consul/docs/upgrading#standard-upgrades

Reply:
i had followed this doc where download is mentioned. https://developer.hashicorp.com/consul/docs/upgrading/instructions/general-process

do you want the entire download section to be removed ?